python/tank/descriptor/descriptor_config.py

```diff
@@ -11,6 +11,7 @@
 from __future__ import with_statement
 
 import os
+import subprocess
```

Comment:
`subprocess.check_output` was introduced in Python 2.7, but we still support 2.6. Please replace this import with a relative import

`from ..util.process import subprocess_check_ouptut`